commands/configure/functions.sh

```diff
@@ -65,22 +65,25 @@ function configure-deis-root {
 }
 
 function configure-registry {
+  if [ ${DEIS_SOURCE} -eq 1 ]; then
+    export DEV_REGISTRY="registry.hub.docker.com"
+    export IMAGE_PREFIX="deis"
+  fi
```

Comment:
If we're provisioning a release, we should always use registry.hub.docker.com as the DEV_REGISTRY.